contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenSource.sol

```diff
@@ -110,45 +77,9 @@ contract NativeTokenSource is
     }
 
     /**
-     * @dev See {TeleporterUpgradeable-receiveTeleporterMessage}.
-     *
-     * Receives a Teleporter message and routes to the appropriate internal function call.
-     */
-    function _receiveTeleporterMessage(
-        bytes32 senderBlockchainID,
-        address senderAddress,
-        bytes memory message
-    ) internal override {
-        // Only allow messages from the destination chain.
-        require(
-            senderBlockchainID == destinationBlockchainID,
-            "NativeTokenSource: invalid destination chain"
-        );
-
-        // Only allow the partner contract to send messages.
-        require(
-            senderAddress == nativeTokenDestinationAddress, "NativeTokenSource: unauthorized sender"
-        );
-
-        // Decode the payload to recover the action and corresponding function parameters
-        (SourceAction action, bytes memory actionData) = abi.decode(message, (SourceAction, bytes));
-
-        // Route to the appropriate function.
-        if (action == SourceAction.Unlock) {
-            (address recipient, uint256 amount) = abi.decode(actionData, (address, uint256));
-            _unlockTokens(recipient, amount);
-        } else if (action == SourceAction.Burn) {
-            uint256 newBurnTotal = abi.decode(actionData, (uint256));
-            _handleBurnTokens(newBurnTotal);
-        } else {
-            revert("NativeTokenSource: invalid action");
-        }
-    }
-
-    /**
-     * @dev Unlocks tokens to recipient.
+     * @dev See {TokenSource-_unlockTokens}
      */
-    function _unlockTokens(address recipient, uint256 amount) private {
+    function _unlockTokens(address recipient, uint256 amount) internal override {
```

Comment:
Calling out that the visibility of `_unlockTokens` and `_handleBurnTokens` is changing from `private` to `internal`. This should be okay, given that we don't expect anything to inherit from these contracts (and if someone does, that's on them), but it is a minor change.